SGHTTPRequest/Core/SGHTTPRequest.m

```diff
@@ -805,4 +839,15 @@ - (BOOL)logResponses {
     return self.logging & SGHTTPLogResponses;
 }
 
+#pragma mark - Network Stub Data Source Injection (UITests support)
```

Comment:
We shouldn't need any of this.  The `@property` in the header file should already autogenerate the storage and the accessors.